src/pages/api/files/file/[...slug].ts

```diff
@@ -46,18 +46,23 @@ export const config = {
   },
 };
 
-async function handlePostRequest(
+async function handlePutRequest(
   req: NextApiRequest,
   token: JWT | null,
   res: NextApiResponse,
 ) {
   const readable = Readable.from(req);
-  const { path = '' } = req.query as { path: string };
-  const url = `${process.env.OPENAI_API_HOST}/v1/files${
-    path && `/${encodeURI(path)}`
-  }`;
+  const slugs =
+    typeof req.query.slug === 'string' ? [req.query.slug] : req.query.slug;
```

Comment:
maybe check that is array?
![image](https://github.com/epam/ai-dial-chat/assets/20163971/338c6840-bc9e-4269-a492-fc7685f6bdff)

```suggestion
  const slugs =
    Array.isArray(req.query.slug) ? req.query.slug : [req.query.slug];
```